src/api/system/system.ts

```diff
@@ -451,7 +451,7 @@ export default class System extends Base {
      * @return {Promise.<void>}
      */
     public releaseExternalProcess(uuid: string): Promise<void> {
-        return this.wire.sendAction('release-external-process', { uuid }).then(() => undefined);
+        return this.wire.sendAction<void>('release-external-process', { uuid }).then(() => undefined);
```

Comment:
<void> => <void, string>

Reply:
It takes an object of {uuid: string} so Identity (default) is correct